app/src/containers/CartPage.tsx

```diff
@@ -179,28 +223,48 @@ class CartPage extends React.Component<RouteComponentProps, CartPageState> {
     return ('');
   }
 
+  handleModalOpen() {
+    this.setState({
+      openModal: true,
+    });
+  }
+
+  handleModalClose() {
+    this.setState({
+      openModal: false,
+    });
+  }
+
+  handleCartSelect(cartData) {
+    this.setState({ cartData });
+  }
+
+  handleCartElementSelect(index) {
+    const { cartsData } = this.state;
+    this.setState({ cartData: cartsData._element[index] });
+  }
+
   render() {
-    const { cartData, isLoading } = this.state;
+    const {
+      cartData, cartsData, isLoading, openModal, multiCartsAvailable,
+    } = this.state;
     const itemDetailLink = '/itemdetail';
+    const cartName = cartData && cartData._descriptor && cartData._descriptor[0].default !== 'true' ? cartData._descriptor[0].name : intl.get('default');
```

Comment:
If multi-cart is not enabled, the page title will still be "Default Cart".
![image](https://user-images.githubusercontent.com/39800563/66933384-03c21880-f007-11e9-8459-0916df747125.png)

This needs to be fixed. If multi-cart is not enabled, then we should fall back to a general title, similar to what we had before. Please use `Your Shopping Cart`:
![image](https://user-images.githubusercontent.com/39800563/66933495-2b18e580-f007-11e9-8fd2-95aad111fc29.png)